CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Minor: Delete all but the last 5 crashdumps on application start. (#4392)
 - Minor: Added `/banid` command that allows banning by user ID. (#4411)
+- Minor: Added ability to reply to message by `Shift + Right Click`ing the username. (#4424)
```

Comment:
```suggestion
- Minor: Added the ability to reply to a message by `Shift + Right Click`ing the username. (#4424)
```
personally I'm fine with the `Click`ing since I commonly `-ing` things